bundle/compliance/cis_aws/rules/cis_1_11/rule.rego

```diff
@@ -0,0 +1,24 @@
+package compliance.cis_aws.rules.cis_1_11
+
+import data.compliance.lib.common
+import data.compliance.policy.aws_iam.data_adapter
+
+default rule_evaluation = true
+
+# Do not setup access keys during initial user setup for all IAM users that have a console password.
+rule_evaluation = false {
+	data_adapter.iam_user.HasLoggedIn
+	key := data_adapter.active_access_keys[_]
+	not key.HasUsed
```

Comment:
not sure how this audits what is described in the rule, can you please explain?

Reply:
> Do not setup access keys during initial user setup for all IAM users that have a console password

So in the following rule I check that:
1. The user has enabled his password as he was logged in.
2. All the activated access keys have been used.

When both are true, the user has a console password and his access keys are used.